build-support/bin/contributors.sh

```diff
@@ -1,74 +0,0 @@
-#!/usr/bin/env bash
-
-REPO_ROOT="$(git rev-parse --show-toplevel)"
-cd "$REPO_ROOT" || exit 1
-
-# shellcheck source=build-support/common.sh
-source "${REPO_ROOT}/build-support/common.sh"
-
-function usage() {
-  echo "Generates contributor lists."
-  echo
-  echo "Usage: $0 (-h|-s revision)"
-  echo " -h           print out this help message"
-  echo " -s           generate the contribution roster since the given revision"
-  echo
-  echo "By default, CONTRIBUTORS.md is re-generated to account for any new"
-  echo "since the last re-generation."
-  echo
-  echo "If \`-s [revision]\` is specified, then the contribution roster since that"
-  echo "revision is output.  This is useful to generate a thank-you list for"
-  echo "release announcements by specifying \`-s [previous release tag]\`."
-
-  if (($# > 0)); then
-    die "$@"
-  else
-    exit 0
-  fi
-}
-
-since=""
-
-while getopts "hs:" opt; do
-  case ${opt} in
-    h) usage ;;
-    s) since="${OPTARG}" ;;
-    *) usage "Invalid option: -${OPTARG}" ;;
-  esac
-done
-
-function contributors() {
-  range="${1:-HEAD}"
-
-  # Include all commits in range but exclude:
-  #  - all commits from the imported zinc tree.
-  #  - commits that are from running the publish goal
-  git log --use-mailmap --format="format:%aN;%s" "${range}" ^imported_zinc_tree |
-    grep -v "pants build committing publish data" |
```

Comment:
I don't see this step in the python version, but I guess that is due to the fact that we don't count the number of contributions, so the individual doing the release will get credit for the release, if nothing else at least, where as before the release process was not included in the count for number of contributions..

Reply:
We don't use the v1 publish goal anymore, so I think this filtering is irrelevant